view/frontend/templates/js/replacejs.phtml

```diff
@@ -1007,24 +1007,28 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
                             if (BoltState.cartRestricted) {
                                 if (popUpOpen) reject(new Error(data.message));
                                 hintBarrier.resolve(hints);
+                                cartBarrier.resolve(cart);
```

Comment:
Where is this `cart` variable defined? I couldn't find it.

Reply:
I ended up redoing this and finding old billingAddress payment only listener and fixing it that way so this is no longer in the PR